name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesTable.java

```diff
@@ -389,13 +390,74 @@ private void addDeltaColumn() // NOSONAR
             if (previous1.isPresent() && !previous2.isPresent())
                 return 1;
 
-            double latestQuote1 = previous1.get().getLeft().getValue() / Values.Quote.divider();
-            double previousQuote1 = previous1.get().getRight().getValue() / Values.Quote.divider();
-            double v1 = (latestQuote1 - previousQuote1) / previousQuote1 * 100;
+            double latestQuote1 = previous1.get().getLeft().getValue();
+            double previousQuote1 = previous1.get().getRight().getValue();
+            double v1 = (latestQuote1 - previousQuote1) / previousQuote1;
 
-            double latestQuote2 = previous2.get().getLeft().getValue() / Values.Quote.divider();
-            double previousQuote2 = previous2.get().getRight().getValue() / Values.Quote.divider();
-            double v2 = (latestQuote2 - previousQuote2) / previousQuote2 * 100;
+            double latestQuote2 = previous2.get().getLeft().getValue();
+            double previousQuote2 = previous2.get().getRight().getValue();
+            double v2 = (latestQuote2 - previousQuote2) / previousQuote2;
+
+            return Double.compare(v1, v2);
+        }));
+        support.addColumn(column);
+    }
+
+    private void addDeltaAmountColumn() // NOSONAR
+    {
+        Column column;
+        column = new Column("6", Messages.ColumnChangeOnPreviousAmount, SWT.RIGHT, 80); //$NON-NLS-1$
```

Comment:
One note: way back, the columns were just ordered. For these historical reasons, some columns have a number as an "id". It was the position of the column in the original set of columns. Today, I give them short readable names. The requirement is that the name must be unique for a set of columns in a view.